investor_wallet_platform_base/models/subscription_request.py

```diff
@@ -67,6 +68,7 @@ def create_comp_sub_req(self, vals):
         subscription_request.send_new_subscription_request_notification_email(
             is_company=False
         )
+        subscription_request._compute_display_national_number()
```

Comment:
Idem.

Reply:
I wasn't sure so I retested, and yes I do. If not, the field National Number will not be displayed when go to the sub request form in the backend.